skunkworks_crow/src/main/java/org/odk/share/bluetooth/BluetoothUtils.java

```diff
@@ -0,0 +1,76 @@
+package org.odk.share.bluetooth;
+
+import android.app.Activity;
+import android.bluetooth.BluetoothAdapter;
+import android.os.Build;
+
+import java.util.UUID;
+
+/**
+ * Basic tools for bluetooth features.
+ *
+ * @author huangyz0918 (huangyz0918@gmail.com)
+ * @since 04/06/2019
+ */
+public class BluetoothUtils {
+
+    // UUID for security bluetooth pairing.
+    public static final UUID SPP_UUID = UUID.fromString("724f00a0-795e-4272-a85a-11075e760e58");
+
+    private BluetoothUtils() {
+
+    }
+
+    private static BluetoothAdapter getBluetoothAdapter() {
+        return BluetoothAdapter.getDefaultAdapter();
+    }
+
+    /**
+     * Checking if current device supports bluetooth.
```

Comment:
"Returns true if the device supports Bluetooth otherwise throws an IllegalStateException"

Reply:
Improved.

Reply:
I think @shobhitagarwal1612 wants to say if this method is used to check whether bluetooth is supported or not, so it is expected to return either tue or false. It shouldn't throw any Exception and if it throws then the caller should be catching or handling it.

Reply:
@huangyz0918 Either add throws in method signature and let its caller handle it or don't throw any exception just return `false`